getting-started/kubernetes/apiserver-preview.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Try out the Calico API server tech-preview
+description: Install the Calico API server on an existing Calico cluster
+canonical_url: '/getting-started/kubernetes/quickstart'
+---
+
+### Big picture
+
+[ **Feature status**: Tech-preview in v3.19 ]
+
+Install the Calico API server on an existing cluster to enable management of Calico APIs using kubectl.
+
+> **Warning!** The Calico API server is a tech preview feature and should not be used in production clusters. It has had limited testing and is liable to change in future releases. Please provide feedback and issue reports via GitHub.
+{: .alert .alert-danger }
+
+### Value
+
+Normally, calicoctl is required in order to manage Calico APIs in the `projectcalico.org/v3` API group. The API server provides a REST API for Calico, and allows management of these APIs using kubectl instead.
+
+### Before you begin
+
+- A cluster with Calico installed using the Kubernetes API data store.
+
+- A machine with `openssl` installed.
+
+### Concepts
+
+#### calicoctl vs kubectl
+
+In previous releases, calicoctl has been required to managed Calico API resources in the `projectcalico.org/v3` API group. The calicoctl CLI tool provides important validation and defaulting on these APIs. The Calico API server performs
+that defaulting and validation server-side, exposing the same API semantics without a dependency on calicoctl.
+
+calicoctl is stil required for the following subcommands:
+
+- [calicoctl node]({{site.baseurl}}/reference/calicoctl/node)
+- [calicoctl ipam]({{site.baseurl}}/reference/calicoctl/ipam)
+- [calicoctl convert]({{site.baseurl}}/reference/calicoctl/convert)
+- [calicoctl version]({{site.baseurl}}/reference/calicoctl/version)
```

Comment:
😢 list (convert is okay, but node and ipam).
`version` is mentioned for completeness? Seems superfluous ...

Reply:
Yeah, hoping we shorten this list over the coming releases. But one step at a time!

Version is listed for completeness, and convert is a command that hopefully we will just remove. It's really the first two.